mlflow/models/utils.py

```diff
@@ -1518,20 +1517,18 @@ def _convert_llm_input_data(data):
     return _convert_llm_ndarray_to_list(data)
 
 
-def _get_temp_file_with_content(file_name: str, content: str, content_format) -> str:
+def _get_temp_file_with_content(temp_dir: str, file_name: str, content: str, content_format) -> str:
```

Comment:
not a blocker: do we still need this function?

```
with open(os.path.join(temp_dir, file_name, content_format), "w") as f:
    f.write(content)
```

seems more straightforward.

Reply:
+ 1 to this. The function doesn't need to exist any more.